open-reac/src/main/resources/openreac/acopf.mod

```diff
@@ -0,0 +1,250 @@
+###############################################################################
+#
+# Copyright (c) 2022 2023 2024, RTE (http://www.rte-france.com)
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
```

Comment:
```suggestion
# SPDX-License-Identifier: MPL-2.0
```
Could you add the license identifier on .mod, .run and .dat files?

Reply:
Done. Ty :)